Libraries/LibJS/Runtime/GlobalObject.cpp

```diff
@@ -572,7 +572,7 @@ JS_DEFINE_NATIVE_FUNCTION(GlobalObject::escape)
     // 2. Let length be the length of string.
     // 5. Let k be 0.
     // 6. Repeat, while k < length,
-    for (auto code_point : TRY_OR_THROW_OOM(vm, utf8_to_utf16(string))) {
+    for (auto code_point : TRY_OR_THROW_OOM(vm, utf8_to_utf16(string)).data) {
```

Comment:
is that thing correctly lifetime extended after the TRY?
Maybe split out the utf16 string into a var before the loop